react/features/toolbox/components/Toolbox.web.js

```diff
@@ -61,6 +69,8 @@ class Toolbox extends Component {
         APP.UI.addListener(
             UIEvents.SHOW_CUSTOM_TOOLBAR_BUTTON_POPUP,
             showCustomToolbarPopup);
+
+        this.props._setDefaultToolbarButtons();
```

Comment:
I second that.

We are very particular:

1. FIXME/TODO comment on the source code implementing the workaround/hotfix. It should explain what the workaround is about, why we chose to have a workaround now, etc.

2. All commits (not just the ones implementing workarounds) must follow a format:
- Subject (1 line)
- Description (1+ paragraphs). In the case of workarounds, it can/should at least contain the text of the FIXME/TODO mentioned in 1.